src/utils/tagObject.ts

```diff
@@ -0,0 +1,40 @@
+const redactableKeys = ["authorization", "x-authorization", "password", "token"];
+const maxDepth = 5;
+
+export function tagObject(currentSpan: any, key: string, obj: any, depth = 0): any {
+  if (depth >= maxDepth) {
+    return;
+  } else {
+    depth += 1;
+  }
+  if (obj === null) {
+    return currentSpan.setTag(key, obj);
+  }
+  if (typeof obj === "string") {
+    let parsed: string;
+    try {
+      parsed = JSON.parse(obj);
```

Comment:
If it's a string, does it need to be parsed?

Reply:
Good question. This is needed to deeply tag nested JSON into separate items, otherwise it just tags an object (ala: `setTag('request.body', { foo: { blah: 'ahhh' }})`, which doesn't display nicely in the app.